modules/codemod/lib/v7/recategorizeIconButtons.ts

```diff
@@ -0,0 +1,192 @@
+   *    - Swap it out for the appropriate mapping
+   */
+  root
+    .find(
+      j.JSXElement,
+      (value: JSXElement) =>
+        value.openingElement.name.type === 'JSXIdentifier' &&
+        (value.openingElement.name.name === importMap.IconButton ||
+          value.openingElement.name.name === styledMap.IconButton)
+    )
+    .forEach(nodePath => {
+      const attrs = nodePath.value.openingElement.attributes;
+
+      const variantProp = attrs?.find(
+        attr => attr.type === 'JSXAttribute' && attr.name.name === 'variant'
+      );
+      // Default IconButton variant is `circle`
+      if (!variantProp) {
+        buttonType = 'TertiaryButton';
+        updateJSXTag(nodePath, 'TertiaryButton');
+        requiredImportSpecifiers.push('TertiaryButton');
+      } else {
+        const isCircleVariant =
+          ((variantProp as JSXAttribute).value as StringLiteral)?.value === 'circle';
+        const isCircleFilledVariant =
+          ((variantProp as JSXAttribute).value as StringLiteral)?.value === 'circleFilled';
+        const isInverseVariant =
+          ((variantProp as JSXAttribute).value as StringLiteral)?.value === 'inverse';
+        const isInverseFilledVariant =
+          ((variantProp as JSXAttribute).value as StringLiteral)?.value === 'inverseFilled';
+
+        if (isCircleVariant) {
+          nodePath.value.openingElement.attributes?.splice(attrs?.indexOf(variantProp)!, 1);
+          buttonType = 'TertiaryButton';
+        } else if (isCircleFilledVariant) {
+          nodePath.value.openingElement.attributes?.splice(attrs?.indexOf(variantProp)!, 1);
+          buttonType = 'SecondaryButton';
+        } else if (isInverseVariant) {
+          buttonType = 'TertiaryButton';
+        } else if (isInverseFilledVariant) {
+          buttonType = 'SecondaryButton';
+        } else {
+          nodePath.value.openingElement.attributes?.splice(attrs?.indexOf(variantProp)!, 1);
+          buttonType = 'TertiaryButton';
+        }
+
+        updateJSXTag(nodePath, buttonType);
+        requiredImportSpecifiers.push(buttonType);
+      }
+    });
+
+  // Find all instances of IconButton within a style function
+  // const StyledIconButton = styled(IconButton) gets renamed to
+  // const StyledIconButton = styled(CorrectButtonMapping)
+  root.find(j.VariableDeclarator).forEach(nodePath => {
+    if (
+      nodePath.value.init?.type === 'CallExpression' &&
+      nodePath.value.init.callee.type === 'CallExpression' &&
+      nodePath.value.init.callee.arguments[0].type === 'Identifier'
+    ) {
+      console.log(nodePath.value.init.callee.arguments[0].name);
+      console.log(buttonType);
+
+      nodePath.value.init.callee.arguments[0].name = buttonType;
+      requiredImportSpecifiers.push(buttonType);
+    }
+  });
+
+  /**
+   * Remove old imports: `IconButton`
+   * Add new required imports
+   */
+  const buttonImports = root.find(j.ImportDeclaration, {
+    source: {value: '@workday/canvas-kit-react/button'},
```

Comment:
```suggestion
    source: {value: (value: string) => value.includes('@workday/canvas-kit-react')},
```

This will allow to change all component even imported from main package